pykern/pkcompat.py

```diff
@@ -99,3 +114,35 @@ def _assert_type(value, typ):
                 type(value),
             ),
         )
+
+
+def _version_ok(major, minor):
```

Comment:
Is there a reason to prefer this over the code below? I've seen more of the code below (not that that means much...)
```python
def _version_ok(major, minor)
    return sys.version_info >= (major, minor)
```

Reply:
Thanks. I didn't know what I was doing.

Reply:
806acaa